#/** ------ 车辆对应的道路状态 ------ */
uint64  u8RoadId
uint32  u4ObjectId
float32 u4DefaultSpeed
float32 u4WaterLevel
float32 u4DistToJunc